inc/menu_put.h

```diff
@@ -18,9 +18,22 @@
 # include "miniRT_config.h"
 # include "render.h"
 # include "vec2.h"
+# ifdef IS_BONUS
```

Comment:
Is implemented as 
# if IS_BONUS==1 in the other files

Reply:
Can change.
Is there a reason for the equals 1?

Reply:
Tested it without == 1, works as well. Can be omitted.

But there is a difference between #if and #ifdef. The first checks if the following statement is true, the latter checks if the entity has been previously defined. It could be that a macro has been defined with # define STH, but STH = false. Then the conditionals would give different results. Better and safer to use the first.

Reply:
Didn't know that! Changed to if - also in bit_field